issues-guide.md

```diff
@@ -0,0 +1,100 @@
+Issues Guide
+=============
+This is a guide for collaborators and non-collaborators on how Github issues
+are handled and processed. 
+
+
+The Sprint
+-----------
+We are on a weekly sprint cycle and every Monday we get together for a meeting
+on Google Hangouts. Currently our meetings are at 12pm PDT / 7pm UTC and the 
+link is always
+posted to the IRC channel. This is where collaborators are assigned tasks and 
+issues to work on for the week. Note that anyone can become a collaborator and 
+all they have to do is contact
+[@jbenet](https://github.com/jbenet) and join in on a sprint meeting. On the 
+rest of the days when we aren't meeting on 
+Google Hangouts we have a "standup" at the same time in the IRC channel. This is
+a time where everyone is on the channel at once so we can sync up and bring up 
+any topics that need to be discussed. 
+
+
+Github Labels
+--------------
+We make extensive use of Github labels to help us and our tools organize issues.
+Here is a rough guide.
+
+__The colors:__  
```

Comment:
Maybe [Atx-style headers][1] so you can access H3 here?  Using an actual header instead of the double-underscore `<strong>` will get you automatic anchors in many Markdown processors (e.g. [GitHub's][2]).

[1]: http://daringfireball.net/projects/markdown/syntax#header
[2]: https://github.com/dylanPowers/ipfs-community/blob/0b5280bd321c2f36c93398ae83b2f26114119d42/issues-guide.md#github-labels